libcloud/test/storage/test_local.py

```diff
@@ -42,6 +42,29 @@
     LocalStorageDriver = None
 
 
+class PickleableAcquireLockInSubprocess:
```

Comment:
I am not sure about this one, for me the local test was failing with "unpickleable" errors, I am not sure how it worked before/on CI? Maybe it's also just a mac problem

Reply:
Yeah, IIRC there is a known issue with some Python 3.x version and pickling in some multiprocessing scenarios on OS X (encounter this issue in a different project).